rock/stream_aligner_test_helpers.rb

```diff
@@ -0,0 +1,96 @@
+# frozen-string-literal: true
+
+# no-doc
+module Syskit
+    module Aggregator
+        # Test helpers for stream aligner
+        #
+        # These create a synchronization by using the `transformer_stream_aligner_port`
+        # for transformer tasks or `stream_aligner_status_port` for pure stream aligner
+        # tasks by checking the time of the processed samples until one matches the
+        # written sample time or the timeout is reached.
+        module TestHelpers
+            # @api public
+            #
+            # Writes a sample to the port and wait until a status reader matching
+            # `latest_time` and `sample_time_field` is found.
```

Comment:
Again. This is describing the steps, not what the method *does*.

The functionality is "Writes a sample to a component's input port until it has been processed by the stream aligner. It is compatible with both pure stream aligner-based component and a transformer-based one"

Reply:
Can you check them again? Am i missing something?